### PR TITLE
volctl: update to 0.9.0.

### PR DESCRIPTION
**Pull request by teldra**
#### Have the results of the proposed changes been tested?
- [x] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
- [x] I built this PR locally for my native architecture, (x86_64-glibc)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] aarch64-musl
  - [x] armv7l
  - [x] armv6l-musl

### CODE REVIEW

File: srcpkgs/volctl/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Template file for 'volctl'
 pkgname=volctl
-version=0.8.2
+version=0.9.0
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-gobject gtk+3 desktop-file-utils python3-cairo python3-yaml python3-click"
-checkdepends="${depends} dbus pulseaudio xvfb-run"
+checkdepends="${depends} dbus pulseaudio xvfb-run python3-wheel python3-pip"
```
Comment (paper42) [inline srcpkgs/volctl/template:8]: they don't have tests, why is this here? can we also remove do_check, because it doesn't do anything?

Comment (teldra) [reply srcpkgs/volctl/template:8]: https://github.com/void-linux/void-packages/pull/28806#discussion_r578131916
This is the reason.
I am open to delete it.
Ping @ericonr

Comment (ericonr) [reply srcpkgs/volctl/template:8]: Feel free to remove, ahesford's complained about me recommending adding checkdepends when tests aren't defined too.

Comment (teldra) [reply srcpkgs/volctl/template]: If make_check=extended is correct, this can be marked as resolved. If I should use make_check=no, let me know.

Comment (ericonr) [reply srcpkgs/volctl/template]: I suppose it'd be `make_check=no`, since there aren't any to be run.

Comment (paper42) [reply srcpkgs/volctl/template]: You should use `make_check=no # no tests, but check fails with a non-zero return code` or something similar

Comment (paper42) [reply srcpkgs/volctl/template]: oops, I was a few seconds late. Anyway, a comment with the reason would be nice.

Comment (teldra) [reply srcpkgs/volctl/template]: @paper42 your are right about a comment. Added.
